roles/openshift_logging_eventrouter/files/eventrouter-dc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  name: logging-eventrouter
+  namespace: logging
+  labels:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+spec:
+  selector:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+  replicas: 2
```

Comment:
https://github.com/heptio/eventrouter/blob/master/yaml/eventrouter-deployment.yaml#L21-L23

I based that number on their template and a comment in it. I didn't find duplicate events in the logs but also, I am not entirely sure how this works exactly